packages/venia-ui/lib/util/images.js

```diff
@@ -57,17 +57,18 @@ export const generateUrlFromContainerWidth = (
     );
```

Comment:
Why isn't the line above also taking ratio into account?

Reply:
Nice question. No reason not to have it. Fixed in b5126ab3509f6266e325fd7179c5d91362759b3e.

Note: I have not gone into the `useProductImageCarousel` talon, `ProductImageCarousel` component, and the components using this component to make the change of sending it as a prop. Felt out of scope for this PR. If someone wants to change this, they can send it as a prop and then as an argument to this function, if not the function uses the default ratio.